src/transports/ws/sws.c

```diff
@@ -759,7 +835,7 @@ static int nn_sws_fail_conn (struct nn_sws *self, int code, char *reason)
         nn_fsm_raise (&self->fsm, &self->done, NN_SWS_RETURN_CLOSE_HANDSHAKE);
     }
 
-    return 0;
+    return;
```

Comment:
No "pedantic" compiler that I'm aware of "prefers" an explicit return at the end of a void function.  If there is a returned value, then that's different.   (Implicit return values of zero are what compilers bitch about.)  I'd prefer to remove it, actually.